src/RA_Dlg_AchEditor.cpp

```diff
@@ -2252,37 +2252,23 @@ void GenerateResizes(HWND hDlg)
     pDlgAchEditorMin.x = windowRect.Width();
     pDlgAchEditorMin.y = windowRect.Height();
 
-    vDlgAchEditorResize.push_back(ResizeContent(hDlg,
-        GetDlgItem(hDlg, IDC_RA_LBX_CONDITIONS), ResizeContent::ALIGN_BOTTOM_RIGHT, TRUE));
-    vDlgAchEditorResize.push_back(ResizeContent(hDlg,
-        GetDlgItem(hDlg, IDC_RA_ACH_GROUP), ResizeContent::ALIGN_BOTTOM, TRUE));
-
-    vDlgAchEditorResize.push_back(ResizeContent(hDlg,
-        GetDlgItem(hDlg, IDC_RA_CHK_ACH_PAUSE_ON_RESET), ResizeContent::ALIGN_RIGHT, FALSE));
-    vDlgAchEditorResize.push_back(ResizeContent(hDlg,
-        GetDlgItem(hDlg, IDC_RA_CHK_ACH_PAUSE_ON_TRIGGER), ResizeContent::ALIGN_RIGHT, FALSE));
-    vDlgAchEditorResize.push_back(ResizeContent(hDlg,
-        GetDlgItem(hDlg, IDC_RA_CHK_ACH_ACTIVE), ResizeContent::ALIGN_RIGHT, FALSE));
-    vDlgAchEditorResize.push_back(ResizeContent(hDlg,
-        GetDlgItem(hDlg, IDC_RA_CHK_SHOW_DECIMALS), ResizeContent::ALIGN_BOTTOM_RIGHT, FALSE));
-
-    vDlgAchEditorResize.push_back(ResizeContent(hDlg,
-        GetDlgItem(hDlg, IDC_RA_ACH_ADDGROUP), ResizeContent::ALIGN_BOTTOM, FALSE));
-    vDlgAchEditorResize.push_back(ResizeContent(hDlg,
-        GetDlgItem(hDlg, IDC_RA_ACH_DELGROUP), ResizeContent::ALIGN_BOTTOM, FALSE));
-    vDlgAchEditorResize.push_back(ResizeContent(hDlg,
-        GetDlgItem(hDlg, IDC_RA_ADDCOND), ResizeContent::ALIGN_BOTTOM, FALSE));
-    vDlgAchEditorResize.push_back(ResizeContent(hDlg,
-        GetDlgItem(hDlg, IDC_RA_DELETECOND), ResizeContent::ALIGN_BOTTOM, FALSE));
-    vDlgAchEditorResize.push_back(ResizeContent(hDlg,
-        GetDlgItem(hDlg, IDC_RA_COPYCOND), ResizeContent::ALIGN_BOTTOM, FALSE));
-    vDlgAchEditorResize.push_back(ResizeContent(hDlg,
-        GetDlgItem(hDlg, IDC_RA_PASTECOND), ResizeContent::ALIGN_BOTTOM, FALSE));
-    vDlgAchEditorResize.push_back(ResizeContent(hDlg,
-        GetDlgItem(hDlg, IDC_RA_MOVECONDUP), ResizeContent::ALIGN_BOTTOM, FALSE));
-    vDlgAchEditorResize.push_back(ResizeContent(hDlg,
-        GetDlgItem(hDlg, IDC_RA_MOVECONDDOWN), ResizeContent::ALIGN_BOTTOM, FALSE));
-
-    vDlgAchEditorResize.push_back(ResizeContent(hDlg,
-        GetDlgItem(hDlg, IDCLOSE), ResizeContent::ALIGN_BOTTOM_RIGHT, FALSE));
+    using AlignType = ResizeContent::AlignType;
+    vDlgAchEditorResize.emplace_back(::GetDlgItem(hDlg, IDC_RA_LBX_CONDITIONS), AlignType::BottomRight, true);
```

Comment:
Skips the copy/move step and just perfectly forwards parameters via parameter pack.